stl/src/filesys.cpp

```diff
@@ -267,9 +267,10 @@ _FS_DLL unsigned long long __CLRCALL_PURE_OR_CDECL _File_size(const wchar_t* _Fn
 // 1908 is leap year #2
 // 1968 is leap year #17
 
-#define WIN_TICKS_PER_SECOND 10000000ULL
 
-#define WIN_TICKS_FROM_EPOCH (((1970 - 1601) * 365 + 3 * 24 + 17) * 86400ULL * WIN_TICKS_PER_SECOND)
+constexpr UINT64 _Win_ticks_per_second = 10000000ULL;
+constexpr UINT64 _Win_ticks_from_epoch = (((1970 - 1601) * 365 + 3 * 24 + 17) * 86400ULL * _Win_ticks_per_second);

```

Comment:
The parentheses at the beginning and the end are unnecessary and should be removed (our convention is to avoid unnecessary parentheses except when precedence is unclear to humans or compiler warnings). They were necessary for macro expansion safety, but fortunately `constexpr` variables don’t have that problem.

Reply:
okay let me update then